datafusion/sqllogictest/test_files/aggregate.slt

```diff
@@ -6296,6 +6299,54 @@ physical_plan
 01)AggregateExec: mode=Single, gby=[], aggr=[count(NULL)]
 02)--DataSourceExec: partitions=1, partition_sizes=[1]
 
+statement count 0
+drop table t;
+
+# test duplicated shema name issue
+
+statement count 0
+create table t (a int) as values (1), (2);
+
+query I
+select count() from t;
+----
+2
+
+query I
+select count(1) * count(2) from t;
```

Comment:
Could you also please add a test that shows just the values of `count(2)`

For example
```sql
select count(1), count(2), count(1) * count(2) from t;
```